bananagram-frontend/src/components/Pages/GridPage.tsx

```diff
@@ -78,6 +78,19 @@ export const GridPage: FC<{
     );
   };
 
+  const setVectors = (pv: Vector, iv: Vector): Vector => {
+    pv = iv;
+    return pv;
+  };
+
+  const handleHome = (): void => {
+    setDimensions(
```

Comment:
It's definitely my fault because I haven't given very good documentation for my vector library so far, but you can just create a new vector by doing `createVector(0, 0)` to make a vector at 0, 0. I'll write up some better documentation though.

Reply:
But here we can just call setOffset and setDimensions with the initial values I think, not sure if that makes sense to you?

Reply:
This was the only syntax I could get to work that fitted in with the usestate dispatch object and the prevstate not documented iso well issue in the 16.8 use state hook

Reply:
I don't think you need to use prevstate here? Because you're just setting the value, it doesn't depend on the previous value. Should just be able to pass in the value you want it to be, which should just be the initialDimensions and initialOffset? I'm on my phone at the moment so I can't run it myself

Reply:
I don't think you should really need that setVectors function though.

Reply:
can see prevstate not a issue as forcing a reset, will try other method without previous state later about 10 30 ish

Reply:
not using prev state now as it is reseting